db/sql/dbsetup.sql

```diff
@@ -222,7 +222,6 @@ CREATE TABLE fhir_endpoints_availability (
     updated_at              TIMESTAMPTZ NOT NULL DEFAULT NOW()
 );
 
-CREATE INDEX fhir_endpoint_url_index ON fhir_endpoints (url);
```

Comment:
Why was this index removed?

Reply:
I deleted this one as I noticed there was already an index made with the fhir_endpoints table url field `CREATE INDEX fhir_endpoints_url_idx ON fhir_endpoints (url);`